src/plugins/opensearch_dashboards_overview/public/components/add_data/add_data.tsx

```diff
@@ -72,7 +72,7 @@ export const AddData: FC<Props> = ({ addBasePath, features }) => {
             <EuiButtonEmpty
               className="osdOverviewDataAdd__actionButton"
               flush="left"
-              href={addBasePath('#/tutorial_directory/sampleData')}
+              href="/app/home#/tutorial_directory/sampleData"
```

Comment:
Any reason, why the `addBasePath` is not working expected ?

Reply:
I'm not sure, but I will check it once and get back to you.
Thanks for the review.

Reply:
Hi @mihirsoni 
From what I understand, `addBasePath` redirects to `localhost:5601`, so the target href has to be updated. We can add `/app/home` to complete the existing href. WDYT?
```
-              href={addBasePath('#/tutorial_directory/sampleData')} 
+              href={addBasePath('/app/home#/tutorial_directory/sampleData/')}
```

[(opensearch_dashboards_overview) getting_started.tsx#L122](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/opensearch_dashboards_overview/public/components/getting_started/getting_started.tsx#L122)
[(opensearch_dashboards_react) overview_page_header.tsx#L109](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/opensearch_dashboards_react/public/overview_page/overview_page_header/overview_page_header.tsx#L109)